edx_adapt/tests/logic_tests.py

```diff
@@ -45,7 +45,9 @@ def _answer_pre_assessment_problems(self, correct=0, attention_question=True):
         """
         pre_assessments = ['Pre_assessment_{}'.format(i) for i in range(0, 14)]
         for i, problem in enumerate(pre_assessments):
```

Comment:
Please use more explicit name instead of `i`. Right now it's hard to comprehend.

Also, please change argument `correct` to more explicit `correct_answers`.